src/data_provider/src/osinfo/sysOsInfoWin.cpp

```diff
@@ -162,6 +162,17 @@ static std::string getRelease(const std::string& build)
     return release;
 }
 
+static std::string getDisplayVersion()
+{
+    std::string display_version;
+    if(IsWindows8OrGreater())
+    {
+        Utils::Registry currentVersion{HKEY_LOCAL_MACHINE, R"(SOFTWARE\Microsoft\Windows NT\CurrentVersion)"};
+        display_version = currentVersion.string("DisplayVersion");
+    }
+    return display_version;
```

Comment:
What happens if `display_version` is not filled? Please try it with an older Windows version to check its behavior.

Reply:
When the `DisplayVersion` field did not exist, it caused an error when trying to collect it.

To fix this problem, I have uploaded a commit by first checking that the `DisplayVersion` field exists and if it doesn't, then assigning it an `unknown value`.